README.rst

```diff
@@ -1,132 +1,200 @@
+
+.. Stop ignoring
+
+-----
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
+
+-----
+
+========
+Features
+========
+
+* Out-of-the-box support for various `popular languages <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__,
+  such as **C/C++**, **Python**, **Javascript**, **CSS**, **Java** and many
+  others with built-in check routines.
+* User-friendly interfaces such as JSON, interactive CLI or any custom format.
+* Plugins for
+  `gedit <https://github.com/coala-analyzer/coala-gedit>`__,
+  `Sublime Text <https://github.com/coala-analyzer/coala-sublime>`__,
+  `Atom <https://github.com/coala-analyzer/coala-atom>`__,
+  `Vim <https://github.com/coala-analyzer/coala-vim>`__ and
+  `Emacs <https://github.com/coala-analyzer/coala-vim>`__.
+* Optimized performace with multi-threading to parallelize the routines - can
+  complete a 26000 line python repository in less than 3 seconds.
+* File caching support - run only on changed files (experimental).
+
+-----
+
+============
+Installation
+============
+
+To install the **latest stable version** run:
+
+.. code-block:: bash
+
+    $ pip3 install coala-bears
 
-                                                         .o88Oo._
-                                                        d8P         .ooOO8bo._
-                                                        88                  '*Y8bo.
-                                          __            YA                      '*Y8b   __
-                                        ,dPYb,           YA                        68o68**8Oo.
-                                        IP'`Yb            "8D                       *"'    "Y8o
-                                        I8  8I             Y8     'YB                       .8D
-                                        I8  8P             '8               d8'             8D
-                                        I8  8'              8       d8888b          d      AY
-         ,gggo,    ,ggggo,    ,gggo,gg  I8 dP    ,gggo,gg   Y,     d888888         d'  _.oP"
-        dP"  "Yb  dP"  "Y8go*8P"  "Y8I  I8dP    dP"  "Y8I    q.    Y8888P'        d8
-       i8'       i8'    ,8P i8'    ,8I  I8P    i8'    ,8I     "q.  `Y88P'       d8"
-      ,d8,_    _,d8,   ,d8' d8,   ,d8b,,d8b,_ ,d8,   ,d8b,       Y           ,o8P
-    ooP""Y8888PP*"Y8888P"   "Y8888P"`Y88P'"Y88P"Y8888P"`Y8            oooo888P"
+|Stable|
 
-.. Stop ignoring LineLengthBear
+To install the latest development version run:
 
-Get *coala* to lint all your languages in your project with one tool and
-config!
+.. code-block:: bash
 
-Demo (Click to View)
---------------------
+    $ pip3 install coala-bears --pre
 
-|asciicast|
+The latest code from the master branch is automatically deployed as the
+development version in PyPI.
```

Comment:
i know i suggested this, but i think it should be ``a development version``.. :( sorry